jest.config.js

```diff
@@ -1,5 +1,5 @@
+const commonConfig = require('@stardust-ui/internal-tooling/jest')
```

Comment:
nit: may be simplified a bit, given that there is only single place where this is used:

```js
const { coverageReporters } = require( ... )
```